src/views/Alteration.vue

```diff
@@ -1,83 +1,105 @@
+
+    <!-- Review and Certify-->
+    <v-slide-x-reverse-transition hide-on-leave>
+      <div v-if="isSummaryMode && showFeeSummary">
+        <header>
+          <h1>Review and Certify</h1>
+        </header>
+
+        <section class="mt-6">
+          Review and certify the changes you are about to make to your company. Certain changes
+          require an Alteration Notice which will incur a $100.00 Fee.
+        </section>
+
+        <AlterationSummary
+          class="mt-10"
           :validate="getAppValidate"
-          :draftCourtOrderNumber="getFileNumber"
-          :hasDraftPlanOfArrangement="getHasPlanOfArrangement"
-          @emitCourtNumber="setFileNumber($event)"
-          @emitPoa="setHasPlanOfArrangement($event)"
-          @emitValid="setValidFileNumber($event)"
+          @haveChanges="onAlterationSummaryChanges()"
         />
 
-        <staff-payment
+        <DocumentsDelivery
           class="mt-10"
           :validate="getAppValidate"
-          @haveChanges="onStaffPaymentChanges()"
+          @valid="setDocumentOptionalEmailValidity($event)"
         />
-      </template>
 
-    </template>
+        <CertifySection
+          class="mt-10"
+          :validate="getAppValidate"
+        />
+
+        <!-- STAFF ONLY: Court Order and Plan of Arrangement -->
+        <template v-if="isRoleStaff">
+          <h2 class="mt-10">3. Court Order and Plan of Arrangement</h2>
+          <p class="my-3 pb-2">
+            If this filing is pursuant to a court order, enter the court order number. If this
+            filing is pursuant to a plan of arrangement, enter the court order number and select
+            Plan of Arrangement.
+          </p>
+
+          <CourtOrderPoa
+            id="court-order"
+            :validate="getAppValidate"
+            :draftCourtOrderNumber="getFileNumber"
+            :hasDraftPlanOfArrangement="getHasPlanOfArrangement"
+            @emitCourtNumber="setFileNumber($event)"
+            @emitPoa="setHasPlanOfArrangement($event)"
+            @emitValid="setValidFileNumber($event)"
+          />
+
+          <StaffPayment
+            class="mt-10"
+            :validate="getAppValidate"
+            @haveChanges="onStaffPaymentChanges()"
+          />
+        </template>
+      </div>
+    </v-slide-x-reverse-transition>
+
+    <!-- Done-->
+    <v-fade-transition>
+      <div v-if="isSummaryMode && !showFeeSummary">
+        <header>
+          <h1>Review and Certify</h1>
+        </header>
+
+        <section class="mt-6">
+          You have deleted all fee-based changes and your company information has reverted to its
+          original state. If you made any non-fee changes such as updates to your Registered
+          Office Contact Information, please note that these changes have already been saved.
+        </section>
+
+        <!-- TODO: on click, return to entity dashboard -->
+        <v-btn large color="primary" id="done-button" class="mt-8" @click="null" >Done</v-btn>
```

Comment:
Only partly implemented in this commit. I will finish in a future commit. I want to get as much of these changes merged ASAP for all of us to build upon.